ftp/src/main/scala/akka/stream/alpakka/ftp/model.scala

```diff
@@ -97,7 +97,8 @@ object RemoteFileSettings {
       host: InetAddress,
       port: Int = DefaultSftpPort,
       credentials: FtpCredentials = AnonFtpCredentials,
-      strictHostKeyChecking: Boolean = true
+      strictHostKeyChecking: Boolean = true,
```

Comment:
Should we deal with the case of strict host key checking in the context of this issue?. On that case, we probably should let specify a `known_hosts` custom file.